docs/user/app_getting_started.md

```diff
@@ -8,62 +8,35 @@ To install the App, please follow the instructions detailed in the [Administrato
-
-#### get-device-status
-
-Get the status of a device in Nautobot.
-
-#### change-device-status
-
-Set the status of a device in Nautobot.
-
-#### get-device-facts
-
-Get detailed facts about a device from Nautobot in YAML format.
-
-#### get-devices
-
-Get a filtered list of devices from Nautobot.
-
-#### get-rack
-
-Get information about a specific rack from Nautobot.
-
-#### get-circuits
-
-Get a filtered list of circuits from Nautobot.
-
-#### get-circuit-connections
-
-For a given circuit, find the objects the circuit connects to.
-
-#### get-circuit-providers
-
-Get a list of circuit providers.
-
-#### about
-
-Provide link for more information on Nautobot Apps.
-
-#### get-manufacturer-summary
-
-Provides summary of each manufacturer and how many devices have that manufacturer.
-
-### Clear
-
-Scroll the chat history out of view.
-
-## Community Commands
-
-The [nautobot-chatops](https://github.com/topics/nautobot-chatops) topic on GitHub provides a list of Nautobot ChatOps compatible plugins that provide additional commands.
\ No newline at end of file
+Each command can be invoked with `help` sub-command to display all sub-commands with the description.
+
+### `/clear` Command
+
+Scroll the chat history out of view. This command has no sub-commands.
+
+### `/nautobot` Command
+
+Interact with Nautobot by utilizing the following sub-commands:
+
+| Command | Arguments | Description |
+| ------- | --------- | ----------- |
+| `about` || Provide a link for more information on Nautobot Apps. |
+| `change-device-status` | `[device-name]` `[status]` | Set the status of a device in Nautobot. |
+| `get-circuit-connections` | `[provider-slug]` `[circuit-id]` | For a given circuit, find the objects the circuit connects to. |
+| `get-circuit-providers` || Get a list of circuit providers. |
+| `get-circuits` | `[filter-type]` `[filter-value]` | Get a filtered list of circuits from Nautobot. |
+| `get-device-facts` | `[device-name]` | Get detailed facts about a device from Nautobot in YAML format. |
+| `get-device-status` | `[device-name]` | Get the status of a device in Nautobot. |
+| `get-devices` | `[filter-type]` `[filter-value]` | Get a filtered list of devices from Nautobot. |
+| `get-interface-connections` | `[filter-type]` `[filter-value-1]` `[filter-value-2]` | Return a filtered list of interface connections based on filter type, `filter_value_1` and/or `filter_value_2`. |
+| `get-manufacturer-summary` || Provides a summary of each manufacturer and how many devices have that manufacturer. |
+| `get-rack` | `[site-slug]` `[rack-id]` | Get information about a specific rack from Nautobot. |
+| `get-vlans` | `[filter-type]` `[filter-value-1]` | Return a filtered list of VLANs based on filter type and/or `filter_value_1`. |
+
+!!! NOTE
+    All sub-commands are intended to be used with the `nautobot` prefix. For example, to retrieve a filtered list of VLANs, use the command `/nautobot get-vlans`.
+
+{%
```

Comment:
I think we'll need to see how it looks when all integrations have content, but I'm betting this will make for a lot of potential scrolling here. Maybe adding each page to the nav in an "Integrations commands" sub folder will be cleaner.

Reply:
Fixed